### ISSUE TITLE
Report of this course design.

### ISSUE THREAD

**Issue by Stlouislee** [Stlouislee/ClassHelper#3]
We should start writing the report.
@fsZhuangB

**Comment by Stlouislee** [comment]
I am considering cutting down the part of e-book management in this program. What's your idea? @fsZhuangB

**Comment by fsZhuangB** [comment]
> I am considering cutting down the part of e-book management in this program. What's your idea? @fsZhuangB

Yes, because of the time limit, I agree we should delete the e-book module.
Or maybe we cam add it another day.

**Comment by Stlouislee** [comment]
Good.
